app/src/pages/inside/stepPage/modals/makeDecisionModal/makeDecisionModal.jsx

```diff
@@ -268,7 +297,8 @@ const MakeDecision = ({ data }) => {
       {
         id: MACHINE_LEARNING_SUGGESTIONS,
         shouldShow: !isBulkOperation,
-        disabled: !isAnalyzerAvailable || true,
+        // disabled: !isAnalyzerAvailable || true,
+        disabled: false,
```

Comment:
remove comment. Check for `isAnalyzerAvailable` should stay in case analyzer is not available